### ISSUE TITLE
Location guard plus has been stopped working since chrome v130

### ISSUE THREAD

**Issue by hamidziaei** [chatziko/location-guard#120]
Hi there, 
I'd like to report you the new extension (guard plus) faced with the problem after the new version 130 of google chrome. 
- OS: Windows 11
- Browser Chrome v130 
I appreciate your attension in advanced
![Screenshot 2024-10-24 111319](https://github.com/user-attachments/assets/f75bbe6a-899b-458a-bf88-3439ed572ac2)

**Comment by terryl213** [comment]
Similar issue here. Since the new version 130 of Google Chrome, it always pop-up the real location instead of the fixed location on the Mac OSX 13.7. The different is that there is no error message poped up on Mac OSX platform. 

OS: MacOS 13.7
Browser: Chrome v130.

Hope that the extension can be updated to compatible with the Chrome v130 and above. 

Thanks.

**Comment by terryl213** [comment]
For the work around purpose, I have migrated to this script: Location Guard Ng (UserScript).

**Comment by hamidziaei** [comment]
I've seen this extension before, but I couldn't install it on the chrome browser

**Comment by nfornovario** [comment]
yeah same here. i just updated to chrome version 130 and now the fixed location doesn't work any longer

please help

**Comment by SukkaW** [comment]
> I've seen this extension before, but I couldn't install it on the chrome browser

Hello, the creator and the current maintainer of the `Location Guard Ng` here.

`Location Guard Ng` is a userscript and requires a userscript manager (TamperMonkey, ViolentMonkey, and AdGuard are the current supported userscript managers).
